test/crio-wipe.bats

```diff
@@ -32,35 +32,24 @@ function run_crio_wipe() {
 # test_crio_wiped_containers checks if a running crio instance
 # has no containers or pods
 function test_crio_wiped_containers() {
-	run crictl pods -v
-	[ "$status" -eq 0 ]
-	[ "$output" == "" ]
-
-	run crictl ps -v
-	[ "$status" -eq 0 ]
-	[ "$output" == "" ]
+	crictl pods -v
+	crictl ps -v
```

Comment:
So, this removes check that the output from these two commands is empty.

This should be
```sh
output=$(crictl pods -v)
[ "$output" = "" ]
# (same for ps)
```

Reply:
fixed!